qemu/tests/block_hotplug.py

```diff
@@ -64,6 +65,7 @@ def unplug_device(vm, get_disk_cmd, device):
         """
         disks_before_unplug = find_disk(vm, get_disk_cmd)
         device.unplug(vm.monitor)
+        time.sleep(5)
```

Comment:
Suggestion 1: Add comment to explain why sleep 5s here.
Suggestion 2: If not sure rescan time for different windows, suggest to execute recan manually in def find_disk() before "wmic diskdrive get index" executed, like rescan_cmd = params.get("disk_rescan_cmd") session.cmd(rescan_cmd).

Reply:
thanks aliang, updated~